eng/common/TestResources/New-TestResources.ps1

```diff
@@ -183,6 +183,9 @@ if ($TestApplicationId -and !$TestApplicationOid) {
     }
 }
 
+# Determine the Azure context that the script is running in.
+$context = Get-AzContext;
```

Comment:
This was hoisted to allow for other steps to potentially take advantage of the context.   At this point, the environment has been chosen and principals have been authenticated.   The resource group has not yet been created, but I don't believe that is impactful to the context.